packages/synapse-react-client/src/components/FullWidthAlert/FullWidthAlert.tsx

```diff
@@ -133,26 +144,43 @@ function FullWidthAlert(props: FullWidthAlertProps) {
         direction={{ xs: 'column', sm: 'row' }}
         alignItems={{ xs: 'start', sm: 'center' }}
         spacing={{ xs: 1, sm: 2 }}
+        gap={{ xs: '8px', sm: 'initial' }}
         display="flex"
         justifyContent="space-between"
       >
         <Box>
-          {title && <AlertTitle>{title}</AlertTitle>}
-          {description}
+          {isMobile ? (
+            <>
+              {title && (
+                <AlertTitle
+                  sx={{ margin: 0, textAlign: { xs: 'center', sm: 'initial' } }}
+                >
+                  {title}
+                </AlertTitle>
+              )}
+            </>
+          ) : (
```

Comment:
This is a bad idea, not everywhere that we use FullWidthAlert do we guarantee that we show a title, or that the title/description by itself is enough for the user to understand what's going on.

Let's revert this part of the change and decide if we should change only the instance of the alert shown in the design, or if we should do a comprehensive review of all places where we use this component.